map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -66,6 +66,14 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
     private var groundOverlayGenerated: Subject<Boolean> = BehaviorSubject.createDefault(false)
     private var mapClicks: Subject<Boolean> = PublishSubject.create()
     private var leaveTourDialog: AlertDialog? = null
+
+    private var searchObject: ArticObject?
```

Comment:
What do you think of moving this into `SearchManager`?

Reply:
`SearchManager` already has a subject that persists the `searchObject`. This field reads the object from the activity's bundle and passes it to `SearchManager` if the value is not null.

Reply:
Then this should really be a function. How about `getLatestSearchObject()`?